### PR TITLE
Handle errors when saving a new user

### PR DESCRIPTION
**Pull request by gouroujo**
return a internal server error and create a user with the result of the saving if everything is ok

### CODE REVIEW

File: examples/server/node/server.js [hunk 1]
```diff
@@ -185,8 +185,9 @@ app.post('/auth/signup', function(req, res) {
       email: req.body.email,
       password: req.body.password
     });
-    user.save(function() {
-      res.send({ token: createJWT(user) });
+    user.save(function(error,result) {
+      if (error) res.status(500).send({ message: error.message });
```
Comment (sahat) [inline examples/server/node/server.js]: Add braces, and put it on a new line.

File: examples/server/node/server.js [hunk 2]
```diff
@@ -185,8 +185,9 @@ app.post('/auth/signup', function(req, res) {
       email: req.body.email,
       password: req.body.password
     });
-    user.save(function() {
-      res.send({ token: createJWT(user) });
+    user.save(function(error,result) {
```
Comment (sahat) [inline examples/server/node/server.js]: Rename error to err, and add a space after comma.

**Comment by sahat** [comment]
I have added code style comments. After that it should be good to merge.

**Comment by gouroujo** [comment]
done. Thanks for this package ;)

**Pull request merged**